src/graphql/resolvers.js

```diff
@@ -70,29 +70,58 @@ module.exports = {
           eventRound: currentRound,
         });
 
+
         // Sum all bets by index
         const rounds = fill(Array(numOfResults), '0');
+        const userRound = fill(Array(numOfResults), '0');
         each(bets, (bet) => {
+          if (userAddress && userAddress.toLowerCase() === bet.betterAddress) {
```

Comment:
i created a helper `toLowerCase` method in `utils/index.js`. please use that one.

Reply:
and technically wont need to check if `userAddress` is defined. `bet.betterAddress` should always be defined. so if userAddress was undefined, youd be checking `undef === bet.betterAddress` which will fail anyways.